README.md

```diff
@@ -16,32 +16,95 @@ Development team:
 | Rui Lopes Gouveia                            | rui.lopesgouveia@heig-vd.ch     | Lindwing   |
 | Alban Favre                                  | alban.favre@heig-vd.ch          | alfavre    |
 
-## Dependencies
+## **Development**
 
+Below you will find all information related on how to get the development environment up and running.
 
-## Build and install
+### Steps
 
-1. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
-   nisi ut aliquip ex ea commodo consequat.
+1. [Install dependencies](#dependencies)
+2. [Build Docker containers](#build-docker-containers)
+3. [Copy secrets](#copy-secrets)
+4. [Run Docker stack](#run-docker-stack)
 
-2. Duis aute irure dolor in reprehenderit in voluptate velit esse
-   cillum dolore eu fugiat nulla pariatur.
 
-3. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui
-   officia deserunt mollit anim id est laborum.
+### Dependencies
 
-## Run
+This project requires:
 
-1. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
-   nisi ut aliquip ex ea commodo consequat.
+- git
+  - https://git-scm.com/downloads
+- Docker (up-to-date version)
+  - https://docs.docker.com/install/
+- Docker-compose
+  - https://docs.docker.com/compose/install/
 
-2. Duis aute irure dolor in reprehenderit in voluptate velit esse
-   cillum dolore eu fugiat nulla pariatur.
+The containers will embed the other dependencies.
 
-3. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui
-   officia deserunt mollit anim id est laborum.
+### Build Docker containers
 
-## Documentation
+1. Clone this repository
+```
+git clone git@github.com:kayoumido/HEIGVD-PRO-A-11.git
+```
+
+2. Go to the appropriate environment folder (ex. ``dev``)
+```
+cd infra/dev
+```
+3. Run docker-compose to build all the container images
+```
+docker-compose build
+```
+4. The build will take some times so go get a coffee.
+
+### Copy secrets
+
+App secrets are obviously not stored in the repo.
+
+For Laravel, place your `.env` in `infra/dev/laravel/`.
+
+### Run Docker stack
+
+To start the containers, run the following (still in the same folder as the `docker-compose.yml`)
+
+```
+docker-compose up # to start the stack
+docker-compose up -d # to start the stack and detach the terminal
+```
+
+Check on which IP belongs to the Web server (look for `IPAddress`)
+```
+docker inspect dev_nginx
+```
+
+Finally, go to http://ipaddr and check everything works nominally.
```

Comment:
Server seems to be running but when I go and check if everything works, I get a `500 Server Error`.

I followed the instructions in the readme